resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +238,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AlwaysShowScrollbars {
+    [DscProperty(Key)] [BinarySettingState] $ShowScrollBars = [BinarySettingState]::KeepCurrentValue
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $ShowBars = [BinarySettingState]::Disabled                
+        } else {
+			$ShowBars = (Get-ItemProperty -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars
+			$currentState.ShowScrollBars = switch ($ShowBars) {
+				1 { [BinarySettingState]::Disabled }               
+				0 { [BinarySettingState]::Enabled }               
+				default { [BinarySettingState]::KeepCurrentValue }
+			}
+				
+		}
+		
+        return $currentState
+    }
+
+    [bool] Test() {
+        $currentState = $this.Get()
+        if ($this.ShowScrollBars -ne [BinarySettingState]::KeepCurrentValue -and $this.ShowScrollBars -ne $currentState.ShowScrollBars) {
+            return $false
+        }
+
+        return $true
+    }
+
+    [void] Set() {
+        if ($this.ShowScrollBars -ne [BinarySettingState]::KeepCurrentValue) {
+            $desiredState = switch ([BinarySettingState]($this.ShowScrollBars)) {
+                Disabled { '1' }
+                Enabled { '0' }
+            }
+
+            if (-not (Test-Path -Path $global:PointerRegistryPath)) {
+                New-Item -Path $global:PointerRegistryPath -Force | Out-Null
+            }
+
+            Set-ItemProperty -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty -Value $desiredState            
+            

```

Comment:
nit: extra line

Reply:
The expaned format is usually preferred for readability, as it increases the clearspace between each idea. I find the condensed format instead to be much more readable, as it displays more data on the screen at once.